src/quo2/components/avatars/wallet_user_avatar.cljs

```diff
@@ -28,21 +28,26 @@
 (defn wallet-user-avatar
   "params, first name, last name, color, size
    and if it's dark or not!"
-  [{:keys [f-name l-name customization-color size]
+  [{:keys [f-name l-name customization-color size monospace? uppercase?]
```

Comment:
What is the `monospace?` prop used for?

Reply:
It is used to render the text with monospace font, because it is required to render `0x` with monospace as per [Figma designs](https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=31009-10341&mode=design&t=s54924DkVVBuQK8e-4)